modules/templates/helper.go

```diff
@@ -376,15 +376,15 @@ func NewFuncMap() []template.FuncMap {
 			// the table is NOT sorted with this header
 			return ""
 		},
-		"RenderLabels": func(labels []*issues_model.Label) template.HTML {
+		"RenderLabels": func(labels []*issues_model.Label, repoLink string) template.HTML {
 			html := `<span class="labels-list">`
 			for _, label := range labels {
 				// Protect against nil value in labels - shouldn't happen but would cause a panic if so
 				if label == nil {
 					continue
 				}
-				html += fmt.Sprintf("<div class='ui label' style='color: %s; background-color: %s'>%s</div> ",
-					label.ForegroundColor(), label.Color, RenderEmoji(label.Name))
+				html += fmt.Sprintf("<a href='%s' class='ui label' style='color: %s; background-color: %s'>%s</a> ",
+					repoLink+"/issues?q=&labels="+strconv.FormatInt(label.ID, 10), label.ForegroundColor(), label.Color, RenderEmoji(label.Name))
```

Comment:
I think it would be a good idea to check first if `strings.HasSuffix(repoLink, "/")`, and append `/` if that's not the case and only then add the rest.
In the worst case, we waste a little bit CPU power.
But we do prevent invalid links with this mechanism.

Reply:
I would argue about this change. You can grep the code base, `RepoLink + "/path..."` is used everywhere. No need to check the last slash. If the `RepoLink` is wrong, there will be a lot of problems already.

Reply:
Okay, then I stand corrected…